Create @DescriptionFile annotation to add larger texts into API reference (#2033)
strimzi/strimzi-kafka-operator: api/src/main/resources/descriptions/io.strimzi.api.kafka.model.KafkaMirrorMakerTls.adoc

> The `tls` property can be used to configure TLS encryption.

Reviewer: ```suggestion
Use the `tls` property to configure TLS encryption.
```

> Use the `tls` property to provide a list of secrets with key names under which the certificates are stored in X.509 format.

Reviewer: ```suggestion
Provide a list of secrets with key names under which the certificates are stored in X.509 format.
```
